Gruntfile.js

```diff
@@ -7,10 +9,10 @@ module.exports = function(grunt) {
   grunt.initConfig(configs);
 
   grunt.registerTask('copy-deps', ['clean', 'copy']);
-  grunt.registerTask('default', ['jshint']);
   grunt.registerTask('build-docs', ['sass', 'cog']);
   grunt.registerTask('publish-docs', ['build-docs', 'gh-pages']);
-  grunt.registerTask('serve', ['sass', 'webpack-dev-server:start', 'watch:sass']);
+  grunt.registerTask('serve',
+                     ['sass', 'webpack-dev-server:start', 'watch:sass']);
   grunt.registerTask('start', ['sass', 'webpack', 'watch:sass']);
-  grunt.registerTask('test', ['karma:run']);
+  grunt.registerTask('test', ['eslint', 'karma:run']);
```

Comment:
it might be better to run these like ``['karma:run', 'eslint']`` because sometimes lint errors can be pedantic and after waiting a long time in the queue for a travis build it would suck for it to die on a missing comma without showing us which tests passed or failed.